__tests__/app.test.js

```diff
@@ -50,10 +50,20 @@ describe('api/articles/:article_id', () => {
               body: 'I find this existence challenging',
               created_at: '2020-07-09T20:11:00.000Z',
               votes: 100,
+              //SQL returns the count as a string
+              comment_count: expect.any(String),
```

Comment:
We should absolutely not be amending previous tests for when new functionality is added. All previous tests should still pass when you add a new part. In this case your test was too brittle, by checking if the object has certain keys allows you room to grow and add more keys in the future.

What if you were to redesign the article to have another key? This test would fail again.

Reply:
I have now separated this into 2 tests, the original and one that checks if it contains a comment_count, I hope this resolves the issue. The test never failed before I changed it, I just didn't realise that this was bad practice.